a4kSubtitles/search.py

```diff
@@ -227,22 +227,63 @@ def __prepare_results(core, meta, results):
                 color_list = group
                 break
 
+    def _filter_name(x):
+        name_diff_ignore = quality + codec + audio + color
+        name_diff_ignore += ["multi", 'multiple', 'sub', 'subs', 'subtitle']
+
+        if x.isdigit():
+            x = str(int(x)).zfill(3)
+        elif x.lower() in name_diff_ignore:
+            x = ''
+        return x.lower()
+
+    def _match_numbers(a, b):
+        offset = 0
+        for s in b:
+            s = core.re.sub(r'v[1-4]', "", s)
+            if s.isdigit() and meta.episode and s.zfill(3) == meta.episode.zfill(3):
+                offset += 0.4
+            elif s.isdigit() and s in a:
+                offset += 0.2
+
+        return offset
+
     def sorter(x):
         name = x['name'].lower()
         nameparts = core.re.split(regexsplitwords, name)
 
+        cleaned_nameparts = list(filter(len, map(_filter_name, nameparts)))
+        cleaned_file_nameparts = list(filter(len, map(_filter_name, meta_nameparts)))
+        found_season, found_episode = core.utils.extract_season_episode(name)
+
+        incorrect_sub_offset = 0
```

Comment:
If we know they are incorrect we should remove them from the list. Better name should be `mismatched_subs_offset`
```suggestion
        mismatched_subs_offset = 0
```

Reply:
Moving them to the bottom of the list I think may help users especially if series has a few subtitles. 
`mismatched_subs_offset` not only moves incorrect one to bottom it also move correct one above. `matching_offset` may be more suitable let me know if you prefer `mismatched_subs_offset`